src/compiler/watchUtilities.ts

```diff
@@ -56,7 +57,8 @@ namespace ts {
             writeFile: host.writeFile && writeFile,
             addOrDeleteFileOrDirectory,
             addOrDeleteFile,
-            clearCache
+            clearCache,
+            realpath: host.realpath && realpath

```

Comment:
maybe this should be called targetpath? I can't tell if it's completely new with this PR or not.

Reply:
that's the existing method on system and other hosts like compiler host.